lib/l10n/intl_es.arb

```
@@ -399,5 +399,42 @@
+  "cancelPendingButton": "CANCELACIÓN PENDIENTE",
+  "acceptCancelButton": "ACEPTAR CANCELACIÓN",
+  "disputeButton": "DISPUTA",
+  "fiatSentButton": "FIAT ENVIADO",
+  "completePurchaseButton": "COMPLETAR COMPRA",
+  "sellBitcoinButton": "VENDER BITCOIN",
+  "buyBitcoinButton": "COMPRAR BITCOIN",
+  "paymentMethodLabel": "Método de Pago",
+  "createdOnLabel": "Creado el",
+  "orderIdLabel": "ID de Orden",
+  "creatorReputationLabel": "Reputación del Creador",
+  "ratingLabel": "Calificación",
```

RuboCop (1.75.5)

[warning] Duplicated key in hash literal.

(Lint/DuplicateHashKey)